data/human/frontier jobs.txt

```diff
@@ -282,20 +282,20 @@ mission "Small Pirate gambling"
 					goto shoot
 				`	(Grab the pot and run!)`
 					goto run
-				`	(Let them fleece me.)`
+				`	(Let them fleece you.)`
```

Comment:
This was correct. Action choices (those in parenthesis) are suppose to be in first person.